BlueMap/Chat/src/main/resources/bluemap-chat.css

```diff
@@ -1,14 +1,22 @@
+/*TODO:
+   - Unable to select text while logged out.
+   */
```

Comment:
Hmhh?

Reply:
Oh I forgot to remove it. That was fixed.